python/ray/autoscaler/updater.py

```diff
@@ -241,7 +251,14 @@ def run(self,
             exit_on_fail=False,
             port_forward=None,
             with_output=False,
+            ssh_options=None,
```

Comment:
can you rename this as "ssh_options_override"

Reply:
Sounds good!